plugins/module_utils/redfish_utils.py

```diff
@@ -3051,3 +3052,62 @@ def get_manager_inventory(self, manager_uri):
 
     def get_multi_manager_inventory(self):
         return self.aggregate_managers(self.get_manager_inventory)
+
+    def set_sessionservice(self, sessionservice_config):
+        result = {}
+        response = self.get_request(self.root_uri + self.session_service_uri)
+        if response['ret'] is False:
+            return response
+        current_sessionservice_config = response['data']
+        payload = {}
+        for property in sessionservice_config.keys():
+            value = sessionservice_config[property]
+            if property not in current_sessionservice_config:
+                return {'ret': False, 'msg': "Property %s in sessionservice_config is invalid" % property}
+            if isinstance(value, dict):
+                if isinstance(current_sessionservice_config[property], dict):
+                    payload[property] = value
+                elif isinstance(current_sessionservice_config[property], list):
+                    payload[property] = list()
+                    payload[property].append(value)
```

Comment:
This is the same as:
```suggestion
                    payload[property] = [value]
```